sdk/storage/azure-storage-blob/azure/storage/blob/_quick_query_helper.py

```diff
@@ -27,25 +31,57 @@ class QuickQueryReader(object):  # pylint: disable=too-many-instance-attributes
 
     def __init__(
         self,
-        client=None,
         name=None,
         container=None,
-        progress_callback=None,
+        errors='strict',
         encoding=None,
-        **kwargs
+        headers=None,
+        response=None
     ):
         self.name = name
         self.container = container
-        self.response_headers = None
-        self.total_bytes = None
+        self.response_headers = headers
+        self.size = None
```

Comment:
>size [](start = 13, length = 4)

just to clarity, total_bytes is describing the entire blob, bytesScanned is also for the number of bytes scanned in the blob